index.html

```diff
@@ -141,6 +155,8 @@ <h2>Registry</h2>
 						<a data-cite=
 						"LONGTASKS-1#performancelongtasktiming"><code>PerformanceLongTaskTiming</code></a>
 					</td>
+                                        <td><code>False</code></td>
+                                        <td>NA</td>
```

Comment:
I think we need to say maxBufferSize is 0 here since the buffered flag is also not supported for long tasks. So we keep no 'internal buffer' (or keep a buffer of size 0).

Reply:
makes sense.  Done.